Commandant/Errors.swift

```diff
@@ -39,7 +39,11 @@ internal func informativeUsageError<T>(keyValueExample: String, option: Option<T
 		description += "]"
 	}
 
-	description += "\n\t\(option.usage)"
+	description += option.usage.componentsSeparatedByString("\n")
```

Comment:
I only found:
```swift
func enumerateLines(body: (line: String, inout stop: Bool) -> ())
```

Looks like it's all side effects… Am I missing something? :confused: